exchanges/ftx/ftx.go

```diff
@@ -1255,20 +1256,17 @@ func (f *FTX) SendAuthHTTPRequest(ctx context.Context, ep exchange.URL, method,
 
 		hmac, err = crypto.GetHMAC(crypto.HashSHA256,
 			[]byte(sigPayload),
-			[]byte(f.API.Credentials.Secret))
+			[]byte(creds.Secret))
 		if err != nil {
 			return nil, err
 		}
 
 		headers := make(map[string]string)
-		headers["FTX-KEY"] = f.API.Credentials.Key
+		headers["FTX-KEY"] = creds.Key
 		headers["FTX-SIGN"] = crypto.HexEncodeToString(hmac)
 		headers["FTX-TS"] = ts
-		if subAccount == "" && f.API.Credentials.Subaccount != "" {
-			subAccount = f.API.Credentials.Subaccount
-		}
-		if subAccount != "" {
-			headers["FTX-SUBACCOUNT"] = url.QueryEscape(subAccount)
+		if creds.SubAccount != "" {
+			headers["FTX-SUBACCOUNT"] = url.QueryEscape(creds.SubAccount)
```

Comment:
This might need some tweaking. I'm getting the following:
```
 rpc error: code = Unknown desc = FTX unsuccessful HTTP status code: 401 raw response: {"success":false,"error":"No such subaccount"}
exit status 1
```
From the following:
```
go run .  --apikey="Lemon" --apisecret="LOL!" --timeout=5m getaccountinfo ftx spot
```

Because you are doing the following:
![image](https://user-images.githubusercontent.com/9261323/156485254-3a884887-97c0-41d7-848f-111888abd383.png)